coredb-operator/src/statefulset.rs

```diff
@@ -182,6 +201,13 @@ pub fn stateful_set_from_cdb(cdb: &CoreDB) -> StatefulSet {
                             docker_setup_env
                             docker_create_db_directories
 
+                            cp -r /tmp/pg_sharedir/* $(pg_config --sharedir)/
+                            cp -r /tmp/pg_pkglibdir/* $(pg_config --pkglibdir)/
+                            chown -R :postgres $(pg_config --sharedir)
+                            chmod -R 2775 $(pg_config --sharedir)
+                            chown -R :postgres $(pg_config --pkglibdir)
+                            chmod -R 2775 $(pg_config --pkglibdir)
```

Comment:
I moved this from the `install-extension.sh` script to here, so thats its next to all the other operator specific permission changes.